packages/cli/generators/project/templates/README.md.ejs

```diff
@@ -1,3 +1,30 @@
 # <%= project.name %>
 
-[![LoopBack](https://github.com/strongloop/loopback-next/raw/master/docs/site/imgs/branding/Powered-by-LoopBack-Badge-(blue)-@2x.png)](http://loopback.io/)
+<% if (project.description) { -%>
+> <%= project.description %>
+<% } -%>
+
+This project was generated with [LoopBack CLI](https://github.com/strongloop/loopback-next/tree/master/packages/cli) version <%= cliVersion %>.
+
+## First steps
+
+### Installation
+
+To proceed with the installation of the dependencies in Loopback you must have `nodejs` >= 10 and `npm` installed
+```sh
+npm install
+```
+> The dependencies are already installed for LoopBack 4 application scaffolded using `lb4` command. If you add other dependencies, please make sure you run `npm i` before starting the application.
+### Run
+To start the project is as easy as this
+```sh
+npm start
+```
+
+## Learn More
+You can learn more in the [LoopBack documentation](https://loopback.io/doc/en/lb4/). The developer guide can be found [here](https://github.com/strongloop/loopback-next/blob/master/docs/site/DEVELOPING.md).
```

Comment:
> You can learn more in the [LoopBack documentation](https://loopback.io/doc/en/lb4/).

To learn more about LoopBack 4, please refer to our [documentation](https://loopback.io/doc/en/lb4/).

Reply:
Done